src/Helper/Dto/Catalog/Product/Attribute/Create.php

```diff
@@ -53,21 +53,13 @@ class Create extends Shared
      */
     public function execute(MageAttribute $attribute, string $locale, int $storeId): CreateDto
     {
-        $dto = (new CreateDto())
-            ->setCode($attribute->getAttributeCode())
+        return (new CreateDto())
+            ->setCode((string)$attribute->getAttributeCode())
```

Comment:
in our weird styles which I cannot remember how we decided on, there is s space after
```suggestion
            ->setCode((string) $attribute->getAttributeCode())
```

Reply:
you are right. I just can't figure out how to explain that to PHPStorm, so it won't undo it.